db/models.py

```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model should be unique. Currently, it is set to 'actors', which could cause a reverse accessor clash if another model uses the same related name. Consider changing it to something like 'movies' to reflect the relationship more accurately.